_topics/en/transaction-origin-privacy.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Transaction origin privacy
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Transaction Relay Policy
+  - Privacy Enhancements
+  - Privacy Problems
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: "Bitcoin Wiki: Privacy---Network analysis"
+      link: https://en.bitcoin.it/wiki/Privacy#Traffic_analysis
+
+## Optional.  Each entry requires "title" and "url".  May also use "feature:
+## true" to bold entry and "date"
+optech_mentions:
+  - title: "Bitcoin Core #18861 won't reply to with txes it hasn't yet announced to the requesting peer"
+    url: /en/newsletters/2020/05/27/#bitcoin-core-18861
+
+  - title: "Transcript: TxProbe: discovering Bitcoin's network topology using orphan transactions"
+    url: /en/newsletters/2019/09/18/#txprobe-discovering-bitcoin-s-network-topology-using-orphan-transactions
+
+  - title: "Bitcoin Core #14897 introduces a semi-random order when requesting transactions"
+    url: /en/newsletters/2019/02/12/#bitcoin-core-14897
+
+  - title: "Bitcoin Core #19109 adds a per-peer rolling bloom filter to track recent tx announcements"
+    url: /en/newsletters/2020/07/22/#bitcoin-core-19109
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Anonymity networks
+    link: topic anonymity networks
+
+  - title: V2 P2P Transport
+    link: topic v2 p2p transport
+
+  - title: Dandelion
+    link: topic dandelion
+
+## Optional.  Force the display (true) or non-display (false) of stub
+## topic notice.  Default is to display if the page.content is below a
+## threshold word count
+#stub: false
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Transaction origin privacy** is the ability to hide the origin of a
+  transaction from network surveillance.
+---
+If an eavesdropper can see every transaction relayed by every node, they
```

Comment:
```suggestion
If eavesdroppers can see every transaction relayed by every node, they
```
so the "they" is grammatically correct -- although I'm aware of the more recent usage, I'm old school, free free to ignore.